eng/pipelines/common/templates/runtimes/send-to-helix-step.yml

```diff
@@ -30,7 +30,7 @@ steps:
   parameters:
     osGroup: ${{ parameters.osGroup }}
     restoreParams: /p:DotNetPublishToBlobFeed=true -restore -projects $(Build.SourcesDirectory)$(dir)eng$(dir)empty.csproj
-    sendParams: ${{ parameters.helixProjectArguments }} /p:TargetOS=${{ parameters.osGroup }} /p:TargetArcitecture=${{ parameters.archType  }} /p:Configuration=${{ parameters.buildConfig }}  /maxcpucount /bl:$(Build.SourcesDirectory)/artifacts/log/SendToHelix.binlog
```

Comment:
Typo - "TargetArcitecture" -> "TargetArc**h**itecture" (missing letter 'h')

Reply:
Ahh, sorry, that's the old version, never mind :-).

Reply:
I think that missing h was why it ran with no tests.